src/test/sectionSpec/fraSpecs.ts

```diff
@@ -6025,7 +6025,7 @@ export const FraSpecs: Record<string, SectionSpec> = {
                 migration: {
                   colNames: ['1990', '2000', '2010', '2015'],
                   calcFormula: `forestOwnership.private_ownership || forestOwnership.public_ownership 
-                    ? extentOfForest.forestArea - (forestOwnership.private_ownership || 0) - (forestOwnership.public_ownership || 0)
+                    ? Math.round((extentOfForest.forestArea - (forestOwnership.private_ownership || 0) - (forestOwnership.public_ownership || 0) + Number.EPSILON) * 100) / 100
```

Comment:
all calculation results must be rounded to 2. not each single formula.
check src/server/controller/cycleData/persistNodeValue/calculateDependantNodes/calculateNode.ts line 37 where results is parsed from ExpressionEvaluator

Reply:
great, wasn't 100% sure if this was what we wanted, ill fix this so